airflow-core/src/airflow/dag_processing/manager.py

```diff
@@ -253,7 +253,7 @@ def run(self):
         By processing them in separate processes, we can get parallelism and isolation
         from potentially harmful user code.
         """
-        from airflow.sdk.execution_time.secrets_masker import reset_secrets_masker
+        from airflow._shared.secrets_masker.secrets_masker import reset_secrets_masker
```

Comment:
If we are using different maskers I don't think we need to reset it at all actually. - we can remove this import and L258